node-js-web-server/server100.js

```diff
@@ -0,0 +1,17 @@
+const http = require('http');
+const path = require('path');
+
+http.createServer(function (request, response) {
+   console.log('request url: ', request.url);
+   
+   var filePath = './public' + request.url;
+   if (filePath == './public/') {
+      filePath = './public/index.html';
+   }
+}).listen(8080);
+console.log('Server is running at http://localhost:8080');
+
+// we can get the extension name from filePath variable with
```

Comment:
We can get the extension name from the `filePath` variable with the `path.extname()` method.
Extract the extension name from `filePath` and store it in a variable named `extName`.